### ISSUE TITLE
Folder location of the settings files and downloaded versions 

### ISSUE THREAD

**Issue by duianto** [noidexe/godot-version-manager#27]
Observed in 
- Godot Version Manager v1.13
- Windows 10 22H2

While testing a Godot issue with the default settings,
I renamed the folder:
from `C:\Users\username\AppData\Roaming\Godot`
to `C:\Users\username\AppData\Roaming\Godot-backup`

When I started the `Godot Version Manager`, then the versions list was empty, and the `Refresh` button said "scraping..."

When I checked the renamed `Godot-backup` folder, I found the folder:
`C:\Users\username\AppData\Roaming\Godot-backup\app_userdata\Godot Version Manager`
among lots of Godot projects, I had created.

### Expected

It would be better if the `Godot Version Manager` settings files and downloaded Godot versions had its own folder, outside the Godot settings folder.

Possibly in:
`C:\Users\username\AppData\Roaming\Godot Version Manager`

**Comment by noidexe** [comment]
Makes sense, I'm not sure how to go about implementing it in 1.x in a backwards-compatible way
